.vscode/settings.json

```diff
@@ -29,5 +29,6 @@
   "npm.packageManager": "yarn",
   "typescript.enablePromptUseWorkspaceTsdk": true,
   "typescript.format.enable": false,
-  "typescript.preferences.quoteStyle": "double"
-}
\ No newline at end of file
+  "typescript.preferences.quoteStyle": "double",
+  "liveServer.settings.port": 5501
```

Comment:
This should be moved to your user settings rather than the workspace settings if possible.

Reply:
@radium-v Thank you - fixed!